apps/cli/src/commands/listApps.ts

```diff
@@ -0,0 +1,20 @@
+import { from, tap } from "rxjs";
+import { withDevice } from "@ledgerhq/live-common/hw/deviceAccess";
+import listApps from "@ledgerhq/live-common/hw/listApps";
+import { deviceOpt } from "../scan";
+
+export default {
```

Comment:
maybe add a description ?

when you run `pnpm run:cli` it shows you the list of available commands and each one of them has a short one liner description

Reply:
updated, I also have updated the `Readme.md` with the new command.